README.md

```diff
@@ -1 +1,84 @@
+- Creating and instantiating classes with attributes
+- Creating class and instance methods within our classes
+- Writing pseudocode and creating tests to drive the creation of our code
+- Importing data from a CSV file
+
+## Primary Requirements
+Create a system that will manage information related to ride sharing. The system should have a way to keep track of drivers, riders and the trips that the drivers take with their riders.
+
+
+#### Driver
+- Each driver should have information uniquely identifying them like ID, name, license plate and vehicle identification number
+- Each vehicle identification number should abide by a specific pattern or length to ensure it is a valid vehicle identification number
+- Given a driver object, you should be able to retrieve the list of trip instances that only this driver has taken
+- Given a driver object, you should be able to retrieve an average rating for that driver based on all trips taken
+- You should be able to retrieve all drivers from the CSV file
+- You should be able to find a specific driver using their numeric ID
+
+#### Rider
+- Each rider should have information uniquely identifying them like ID, name and phone number
+- Given a rider object, you should be able to retrieve the list of trip instances that only this rider has taken
+- Given a rider object, you should be able to retrieve the list of all previous driver instances (through the trips functionality built above)
+- You should be able to retrieve all riders from the CSV file
+- You should be able to find a specific rider using their numeric ID
+
+#### Trip
+- Each trip should have information uniquely identifying it like ID, rider ID, a driver ID, date, rating
+- Each rating should be within an acceptable range (1-5)
+- Given a trip object, you should be able to retrieve the associated driver instance through the driver ID
+- Given a trip object, you should be able to retrieve the associated rider instance through the rider ID
+- You should be able to find all trip instances for a given driver ID
+- You should be able to find all trip instances for a given rider ID
+- You should be able to retrieve all trips from the CSV file
+
+## Getting Started
+This is a level 3, individual project.
+
+We will use the same project structure we used for the previous project. Classes should be in files in the `lib` folder, and tests should be in files in the `specs` folder. You should utilize a spec helper file. You will run tests by executing the `rake` command, as configured in a Rakefile. You should have **95% code coverage** using simplecov.
+
+The `support` folder contains CSV files which will drive your system design. Each CSV corresponds to a different type of object _as well as_ creating a relationship between different objects.
+
+## Baseline
+### Discussion
+#### Phase 1
+Let's start by **individually** creating the list of nouns and verbs that you would utilize to describe this problem.
```

Comment:
I don't think there's any way around it really, but we do effectively give them all of the nouns in the requirements above.